Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["JuliaStats"]
 version = "0.19.1"
 
 [deps]
+Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
```

Comment:
`Distributed` is a Test dependency, we should add it in the `extras` and `targets` only